File: server/src/main/java/hm/binkley/labs/beer/BeerCommandLineRunner.java
package hm.binkley.labs.beer;

import lombok.RequiredArgsConstructor;
import org.springframework.boot.CommandLineRunner;
import org.springframework.stereotype.Component;

import java.util.stream.Stream;

import static java.lang.System.out;

@Component
@RequiredArgsConstructor
public final class BeerCommandLineRunner
        implements CommandLineRunner {
    private final BeerRepository repository;

    @Override
    public void run(final String... args) {
        Stream.of("Kentucky Brunch Brand Stout", "Lawnmower", "Budweiser",
                "Coors Light").
                forEach(name -> repository.save(new Beer(name)));

        repository.findAll().
                forEach(out::println);
    }
}

File: server/src/main/kotlin/hm/binkley/labs/beer/BeerCommandLineRunner.kt
package hm.binkley.labs.beer

import org.springframework.beans.factory.annotation.Autowired
import org.springframework.boot.CommandLineRunner
import org.springframework.stereotype.Component

@Component
class BeerCommandLineRunner @Autowired constructor(
        private val repository: BeerRepository)
    : CommandLineRunner {
    override fun run(args: Array<String>) {
        repository.save(Beer(null, "Kentucky Brunch Brand Stout", 94))
        repository.save(Beer(null, "Lawnmower", 88))
        repository.save(Beer(null, "Budweiser", 42))
        repository.save(Beer(null, "Coors Lite", 54))
    }
}
